Adapt Serverless Tier in Developer Guide (#11125)
pingcap/docs: develop/dev-guide-build-cluster-in-cloud.md

## Step 1. Create a free cluster

1. If you do not have a TiDB Cloud account, click [here](https://tidbcloud.com/free-trial) to sign up for an account.

2. [Log in](https://tidbcloud.com/) to your TiDB Cloud account.

3. On the **Clusters** page, click **Create Cluster**.

4. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, and then select the region where you want to create your cluster.

    Your TiDB Cloud cluster will be created in approximately 30 seconds.

> 6. After your TiDB Cloud cluster is created, click **Security Settings**. In the **Security Settings** dialog box, set a root password to connect to your cluster, and then click **Apply**. If you do not set a root password, you cannot connect to the cluster.

Reviewer: ```suggestion
6. After your TiDB Cloud cluster is created, click **Security Settings**. In the **Security Settings** dialog box, set a root password to connect to your cluster, and then click **Submit**. If you do not set a root password, you cannot connect to the cluster.
```

> 7. Then click **Connect**. A connection dialog box is displayed. Under **Connect with a SQL Client** in the dialog, click the tab of your preferred connection method, and then save the corresponding connection string. The following section uses MySQL client as an example.

Reviewer: ```suggestion
7. Click **Connect**. A connection dialog box is displayed. Under **Connect with a SQL Client** in the dialog, click the tab of your preferred connection method, and then save the corresponding connection string. The following section uses MySQL client as an example.
```

    {{< copyable "shell-regular" >}}

    ```shell
    mysql --connect-timeout 15 -u '<prefix>.root' -h <host> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/cert.pem -p
    ```

> > **Note:**
> >
> > - Use TLS connections here is [required](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters).
> > - If you are having issues with connecting, you can read TiDB Cloud [Serverless Tier Clusters](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters) for more information.

Reviewer 2: ```suggestion
<CustomContent platform="tidb">

> **Note:**
>
> - The preceding example uses the TLS connection, which is [required](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters) when connecting a Serverless Tier cluster.
> - If you encounter problems when connecting to a Serverless Tier cluster, you can read [Secure Connections to Serverless Tier Clusters](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters) for more information.

</CustomContent>

<CustomContent platform="tidb-cloud">

> **Note:**
>
> - The preceding example uses the TLS connection, which is [required](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md) when connecting a Serverless Tier cluster.
> - If you encounter problems when connecting to a Serverless Tier cluster, you can read [Secure Connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md) for more information.

</CustomContent>
```

3. Fill in the password to sign in.

> ## Step 3. Run a SQL

Reviewer 2: ```suggestion
## Step 3. Execute a SQL statement
```

> Let's try to run your first SQL on TiDB Cloud.

Reviewer 2: ```suggestion
Let's try to execute your first SQL statement on TiDB Cloud.
```

```sql
SELECT 'Hello TiDB Cloud!';
```

> You will see output like this.

Reviewer 2: ```suggestion
Expected output:
```

```sql
+-------------------+
| Hello TiDB Cloud! |
+-------------------+
| Hello TiDB Cloud! |
+-------------------+
```

> If you see output like this. **_CONGRATULATIONS_**! You have successfully run your own SQL on TiDB Cloud.

Reviewer 2: ```suggestion
If your actual output is similar to the expected output, congratulations, you have successfully execute a SQL statement on TiDB Cloud.
```

    {{< copyable "shell-regular" >}}

    ```shell
    mysql --connect-timeout 15 -u '<prefix>.root' -h <host> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/cert.pem -p
    ```

<CustomContent platform="tidb">

> **Note:**
>

> > - The preceding example uses the TLS connection, which is [required](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters) when connecting a Serverless Tier cluster.

Reviewer: ```suggestion
> - When you connect to a Serverless Tier cluster, you must [use the TLS connection](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters). 
```

> - If you encounter problems when connecting to a Serverless Tier cluster, you can read [Secure Connections to Serverless Tier Clusters](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters) for more information.

</CustomContent>

<CustomContent platform="tidb-cloud">

> **Note:**
>

> > - The preceding example uses the TLS connection, which is [required](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md) when connecting a Serverless Tier cluster.

Reviewer: ```suggestion
> - When you connect to a Serverless Tier cluster, you must [use the TLS connection](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).
```
